# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/mp4parse_capi/tests/test_fragment.rs
# Commit: 9493029c76b9
# Full Hash: 9493029c76b93861ca10dbfb29d4178773144416
# Author: zaggy1024 <Zaggy1024@gmail.com>
# Date: 2023-01-28 21:11:06
# Regressor Bug: 1788119
# File Overlap Count: 1
# Overlapping Files: Cargo.lock
# Description:
#   Bug 1788119 - Part 4 - Update mp4parse-rust for AVIS support. r=kinetik,glandium,supply-chain-reviewers
#   
#   Depends on D156652
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D156653
# ==============================================================================

diff -r 7d5cc211e3f5 -r 9493029c76b9 third_party/rust/mp4parse_capi/tests/test_fragment.rs
--- a/third_party/rust/mp4parse_capi/tests/test_fragment.rs	Fri Jan 27 19:54:29 2023 +0000
+++ b/third_party/rust/mp4parse_capi/tests/test_fragment.rs	Fri Jan 27 19:54:29 2023 +0000
@@ -1,4 +1,3 @@
-extern crate mp4parse_capi;
 use mp4parse_capi::*;
 use std::io::Read;
 